### PR TITLE
change user_extra to user_extras

### PR DESCRIPTION
**Pull request by prometheanfire**
Connects #1380

### CODE REVIEW

**Comment by rackernexus** [comment]
If you want to double-check your formatting, check [the preview](https://staging.developer.rackspace.com/staging.horse/build-a9f52719cd/docs/private-cloud/rpc/release-notes/).

**Comment by sigmavirus24** [comment]
Needs an issue that it's linked back to

File: scripts/test-upgrade.sh [hunk 1]
```diff
@@ -28,13 +28,13 @@ RPCD_DIR="$BASE_DIR/rpcd"
 #        them to the new location.
 "${BASE_DIR}"/scripts/migrate-yaml.py --for-testing-take-new-vars-only \
     --defaults "${RPCD_DIR}"/etc/openstack_deploy/user_rpco_variables_defaults.yml \
-    --overrides /etc/openstack_deploy/user_extra_variables.yml \
+    --overrides /etc/openstack_deploy/user_extras_variables.yml \
     --output-file /etc/openstack_deploy/user_rpco_variables_overrides.yml
 "${BASE_DIR}"/scripts/migrate-yaml.py --for-testing-take-new-vars-only \
   --defaults "${RPCD_DUR}"/etc/openstack_deploy/user_osa_variables_defaults.yml \
```
Comment (alextricity25) [inline scripts/test-upgrade.sh]: Typo: s/DUR/DIR

**Comment by alextricity25** [comment]
We also need to copy over the /opt/rpc-openstack/rpcd/etc/openstack_deploy/user_{rpc,osa}_defaults.yml files over to /etc/openstack_deploy/ as part of this process.
Similar to what is done here:
https://gist.github.com/mattt416/f013453b89addcfd808a3dfd760438ac#file-test-upgrade-sh-L21

**Comment by prometheanfire** [comment]
the copying of files should already be done by deploy.sh iirc

**Comment by prometheanfire** [comment]
copied anyway, doesn't hurt

**Comment by rackernexus** [comment]
:eyes: [Preview](https://staging.developer.rackspace.com/staging.horse/build-09d8a21463/docs/private-cloud/rpc/release-notes/)

**Comment by alextricity25** [comment]
:+1:

**Comment by sigmavirus24** [comment]
Is there are card that this is connected to yet?

**Comment by b3rnard0** [comment]
@prometheanfire Going forward, all pull requests need to have a connecting parent issue. That way we can prioritize and approve the issue and have visibility around it (and provide help on it if needed).

That being said, is this a blocker for an existing approved/in progress card? Do we need to expedite the issue?

**Comment by sigmavirus24** [comment]
@b3rnard0 This typo prevents `test-ugprade.sh` from running. It *may* be a blocker, but no one has complained yet which leads me to believe no one is actually blocked by this.

**Comment by b3rnard0** [comment]
@prometheanfire Still need an issue for this pull request as we need this for r13.1.0rc1.

**Comment by sigmavirus24** [comment]
👎 the commit message only describes a portion of the change and doesn't connect back to the original issue

**Comment by rackernexus** [comment]
Your content preview [is now ready](https://staging.developer.rackspace.com/staging.horse/build-31ca0b7a88/docs/private-cloud/release-notes/). :bow:

**Comment by sigmavirus24** [comment]
Tested and works well. 👍

**Comment by major** [comment]
Looks good to me pending Jenkins. :+1:

**Comment by prometheanfire** [comment]
ok, merging with the two approvals from @sigmavirus24 and @major

**Pull request merged**